src/nodes/Table/TableCell.js

```diff
@@ -30,4 +32,32 @@ export default TableCell.extend({
 			},
 		}
 	},
+
+	addProseMirrorPlugins() {
+		return [
+			new Plugin({
+				props: {
+					handlePaste: (view, event, slice) => {
```

Comment:
```suggestion
					// Special-treat empty lines in pasted content to prevent jumping out of cell
					handlePaste: (view, event, slice) => {
```

A comment would be helpful here to easily grasp later what this whole custom paste handler is about.